File: app/src/main/java/com/egecius/demo_androidplatform/ViewStubActivity.java
package com.egecius.demo_androidplatform;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.ViewStub;

public class ViewStubActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_view_stub);

        inflateIntoViewStub();
    }

    private void inflateIntoViewStub() {
        ViewStub viewStub = findViewById(R.id.my_view_stub);
        viewStub.setLayoutResource(R.layout.layout_to_inject);
        viewStub.inflate();
    }
}

File: app/src/main/java/com/egecius/demo_androidplatform/ViewStubActivity.kt
package com.egecius.demo_androidplatform

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.ViewStub

class ViewStubActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_view_stub)

        inflateIntoViewStub()
    }

    private fun inflateIntoViewStub() {
        val viewStub = findViewById<ViewStub>(R.id.my_view_stub)
        viewStub.layoutResource = R.layout.layout_to_inject
        viewStub.inflate()
    }
}